ruff_cli/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "ruff_cli"
+version = "0.0.220"
+authors = ["Charlie Marsh <charlie.r.marsh@gmail.com>"]
+edition = "2021"
+rust-version = "1.65.0"
+documentation = "https://github.com/charliermarsh/ruff"
+homepage = "https://github.com/charliermarsh/ruff"
+repository = "https://github.com/charliermarsh/ruff"
+readme = "../README.md"
```

Comment:
@messense - This was also necessary. As-is, `ruff-0.0.220.dist-info/METADATA` was missing the README contents.